### PR TITLE
Updated cr-toast properties

### PR DESCRIPTION
**Pull request by simonhong**
fix https://github.com/brave/brave-browser/issues/38097

Updated background/foreground/shadow props to cr-toast used by bookmarks panel

Resolves

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by simonhong** [comment]
@fallaciousreasoning With this PR, our shadow prop is not applied to `cr-toast`.
Am I missing something for this overriding? Need your help. Thanks!

**Review by fallaciousreasoning — APPROVED** [review]

File: ui/webui/resources/polymer_overriding.ts [hunk 1]
```diff
@@ -366,3 +367,6 @@ export function html(strings, ...values) {
   htmlStrings.raw = htmlStrings
   return polymerHtml(htmlStrings).content.cloneNode(true)
 }
+
+// Overrides for all pages
+RegisterStyleOverrideName('cr-toast', 'cr-toast-override')
```
Comment (fallaciousreasoning) [inline ui/webui/resources/polymer_overriding.ts:372]: This seems to be the only place `RegisterStyleOverrideName` is used, so I wouldn't be surprised if it doesn't work....

could you convert to using

```
RegisterStyleOverride('cr-toast', html`<style>...</style>`)
```?

Comment (fallaciousreasoning) [reply ui/webui/resources/polymer_overriding.ts:372]: Alternatively, you might need to include the `cr_toast_override` in `cr_toast` via a patch.

Comment (simonhong) [reply ui/webui/resources/polymer_overriding.ts:372]: Unfortunately, `RegisterStyleOverride` also doesn't work.
```
RegisterStyleOverride(
  'cr-toast',
  html`
    <style>
      :host {
        box-shadow: 0px 4px 8px -1px #0000001A !important;
      }
    </style>
  `
)
```
and adding this to `cr_toast.css` doesn't work
` #import=//resources/brave/overrides/cr_toast_override.css.js`
Maybe I'm missing something or `cr_taost` is lit element?

Comment (simonhong) [reply ui/webui/resources/polymer_overriding.ts:372]: I'm thinking to update only bg/fg colors of toast w/o changing shadow. it's beyond my knowledge.

**Review by iefremov — APPROVED** [review]

File: ui/webui/resources/polymer_overriding.ts (hunk 1 above)
Comment (fallaciousreasoning) [reply ui/webui/resources/polymer_overriding.ts]: Oh... `cr-toast` is indeed a `LitElement`. That's going to make things a bit more complicated. I've just landed a commit in `cr126` which let's us override styles in Lit.

https://github.com/brave/brave-core/pull/23525

So yeah, unfortunately I think we won't be able to override until `cr126` is in.

**Pull request merged**